src/clojure_tic_tac_toe/input_output.clj

```diff
@@ -6,8 +6,31 @@
 (defn display-introduction []
   (println "This is a Tic Tac Toe program.\n"))
 
-(defn display-instructions []
-  (println (join-lines ["To enter a move, type a number from 1-9."
+(defn display-game-mode-instructions []
+  (println (join-lines ["Please enter one of the following:"
+                        "- \"h\" to play another person\n"])))
+
+(defn display-angle-bracket []
+  (print "> ")
+  (flush))
+
+(defn- get-input []
+  (keyword (str/trim (read-line))))
+
+(defn get-game-mode []
+  (get-input))
```

Comment:
I'm wondering if it's better to combine some of these other functions and create this function `get-game-mode`(whether you choose to create it here or back in game_setup) instead of having `set-up-game` know about three different functions to call on. I'm not sure if you've read this blog post yet but we want to try and achieve something like what's in the blog post here: https://8thlight.com/blog/dave-moore/2014/07/25/abstraction.html.